lib/services/oauth/oauth_server.go

```diff
@@ -64,6 +67,20 @@ func GetClientStore() *pg.ClientStore {
 }
 
 func passwordAuthorizationHandler(ctx context.Context, clientID, email string, password string) (string, error) {
-	// TODO: Implement the logic in Sign In task (#26)
-	return "1", nil
+	db := database.GetDB()
+
+	var user models.User
+	result := db.Where("email = ?", email).First(&user)
+	if result.Error != nil {
+		log.Error(result.Error)
+		return "", result.Error
+	}
```

Comment:
```suggestion
	err := db.Where("email = ?", email).First(&user).Error
	if err != nil {
		log.Error(err)
		return "", err
	}
```

No need to apply this suggestion, just for info: a more "Go" approach would be to directly query the `.Error` so you don't need to reach `result.Error` three times :)